### PR TITLE
duplicate archive URL for bazel to retry in case of download issues

### PR DESCRIPTION
**Pull request by gennadiycivil**
Testing potentially having Bazel retry download errors with the same URL

### CODE REVIEW

File: head_sync.py [hunk 1]
```diff
@@ -44,7 +44,7 @@ def workspace_rule(self):
     request = http.request('GET', url,
                            headers = { 'User-Agent': 'Workspace Updater' })
     sha256 = hashlib.sha256(request.data).hexdigest()
-    return HTTP_ARCHIVE_TEMPLATE.format(project.name, url, date, project.repo,
+    return HTTP_ARCHIVE_TEMPLATE.format(project.name, url,url, date, project.repo,
```
Comment (derekmauro) [inline head_sync.py]: Nit: Space after the first `url,`

Comment (derekmauro) [inline head_sync.py]: You should put a comment explaining this strange hack so the next person understands why the URL is listed twice.

Comment (gennadiycivil) [reply head_sync.py]: done

File: head_sync.py [hunk 2]
```diff
@@ -11,7 +11,7 @@
 
 HTTP_ARCHIVE_TEMPLATE = """    http_archive(
       name = "{}",
-      urls = ["{}"],  # {}
+      urls = ["{}","{}"],  # {} use the same urL twice to trick bazel into re-trying if connection fails
```
Comment (derekmauro) [inline head_sync.py]: urL -> URL

**Comment by gcivil-nyu** [comment]
@googlebot I consent.

**Pull request merged**